docs/fleet/_index.md

```diff
@@ -24,14 +24,39 @@ You can manage and control your fleet of {{< glossary_tooltip term_id="machine"
 
 To organize your fleet you use
```

Comment:
```suggestion
To organize your fleet you use:
```
[nit] Sorry for drive-by....I've been wanting to add this colon but see that this PR is in the works so putting it in here instead of making separate PR!